spack-environments/care/spack.yaml

```diff
@@ -0,0 +1,79 @@
+##############################################################################
+# Copyright (c) 2020-2021, Lawrence Livermore National Security, LLC and
+# RADIUSS Stack project contributors. See the LICENSE file for details.
+#
+# SPDX-License-Identifier: MIT
+##############################################################################
+
+spack:
+  #[general--]
+  include:
+  - ../radiuss/$SYS_TYPE
+  concretization: separately
+  view: false
+  #[--general]
+
+  #[config-override--]
+  config:
+    install_tree:
+      root: /usr/workspace/radiuss/install/care-stack
+      padded_length: 128
+      projections:
+        all: '{architecture}/{compiler.name}-{compiler.version}/{name}-{version}-{hash}'
+    misc_cache: $spack/.misc_cache
+    test_stage: $spack/.test_stage
+    'build_stage:':
+    - $spack/var/spack/stage
+  #[--config-override]
+
+  mirrors:
+    mirror: file:///usr/workspace/radiuss/mirrors/llnl-stack
+
+  #[definitions--]
+  definitions:
+  - compilers: [] # ['%gcc@8.1.0', '%gcc@8.3.1', '%clang@10.0.0']
+    when: env.get("SYS_TYPE", "") == "toss_3_x86_64_ib"
+  - compilers: ['%gcc@8.3.1'] # ['%xl@16.1.1.7']
+    when: env.get("SYS_TYPE", "") == "blueos_3_ppc64le_ib_p9"
+  #[definitions--]
+  - projects:
+    when: env.get("SYS_TYPE", "") == "toss_3_x86_64_ib"
+  - projects:
+    - care+cuda cuda_arch=70 +allow-unsupported-compilers ^cuda@10.1.168 ^cmake@3.14.5
+    when: env.get("SYS_TYPE", "") == "blueos_3_ppc64le_ib_p9"
+  - dependencies: []
+    #- hdf5@1.8.21~shared #axom requires shared variant
+  #[--definitions]
+
+  specs:
+  - matrix:
+    - [$projects, $dependencies]
+    - [$compilers]
+
+  gitlab-ci:
+    script:
+    - . ${CHILD_SPACK_PATH}/share/spack/setup-env.sh
+    - cd ${SPACK_CONCRETE_ENV_DIR} && spack env activate --without-view .
+    - spack -d ci rebuild
+
+    mappings:
+      ## quartz
+      ## [runner-mapping--]
+      #- match:
+      #  - target=broadwell
+      #  runner-attributes:
+      #    tags: [quartz, shell]
+      ## [--runner-mapping]
```

Comment:
I think your intent is not to have a quartz pipeline. But in fact the quartz generation still happens, so it may fail in the end.
I suggest commenting out the quartz jobs in the CI for now, in an isolated commit that we can revert later on.